src/common/api_impl.h

```diff
@@ -160,8 +161,9 @@ class DeviceDescription {
 
 private:
   int client_id_;
-  std::string kind_string_ = "wormhole";
-  std::string debug_string_ = "debug_string";
+  std::string kind_string_ = "TTDevice";
+  // TODO should not be hardcoded
+  std::string arch_string_ = "Wormhole";
```

Comment:
It's not a change that impacts anything in pjrt plugin, it was set to dummy value anyway. When you print `jax.Array.device` we were getting `debug`. Now we get `TTDevice(Wormhole)`

Reply:
Oh, I see.

This is not that deep, this is just vaguely defined test. If I change it to

```python
def test_to_device():
    cpu_array = random_input_tensor((32, 32))
    device = jax.devices("tt")[0]  # explicitly use "tt" backend
    tt_array = jax.device_put(cpu_array, device)
    assert tt_array.device.device_kind == "TTDevice"  ##  "TTDevice" instead of "wormhole"
```

it passes. 

Due to change in `tests/conftest.py` I had to make, _order_ of devices registered with jax is now `[cpu, tt]` and previously it was `[tt, cpu]`. I am saying that because doing `jax.devices()` with no arg will return the firstly registered one, which shouldn't be relied upon. We should explicitly say which backend.

String change is not that big of a deal. I will still need to change the `jax.devices("tt")[0] ` part, regardless of if I revert PJRT changes or not. 

Do you still feel like I should revert it?

Reply:
I see, you are right it makes sense to update the test and leave the changes. I would still open an issue for this:
> We should understand better how these are used for, [here](https://github.com/openxla/xla/blob/2da0f3ed4ba9cb4159f9f2bd6f20b32c71ed6b97/xla/pjrt/pjrt_device_description.h#L67) is how XLA defines these.

and add a TODO comment above your changes in `api_impl.h` to rethink those, with a link to that issue.